templates/default/service-usbvm

```diff
@@ -0,0 +1,61 @@
+{
+  "uuid": "00000000-0000-0000-0000-000000000003",
+  "type": "usbvm",
+  "name": "USB",
+  "stubdom": "true",
+  "slot": "-1",
+  "hidden": "true",
+  "start_on_boot": "true",
+  "start_on_boot_priority": "10",
+  "shutdown-priority": "-15",
+  "hidden-in-ui": "false",
+  "measured": "true",
+  "s3-mode": "restart",
+  "image_path": "plugins\/serviceimages\/citrix.png",
+  "argo-firewall-rules": {
+    "0": "0:12222 -> myself:2222",
+    "1": "myself -> 0:7777",
+    "2": "my-stubdom -> 0:5100"
+  },
+  "policies": {
+    "audio-access": "false",
+    "audio-rec": "false",
+    "cd-access": "false",
+    "cd-rec": "false",
+    "modify-vm-settings": "false"
+  },
+  "config": {
+    "notify": "dbus",
+    "debug": "true",
+    "pae": "true",
+    "acpi": "true",
+    "virt-type": "hvm",
```

Comment:
What's the motivation for HVM vs PV?

Reply:
With meltdown/spectre, the preferred order of VM types is PVH > HVM > PV.  PVH can't do PCI passthrough, so I used HVM.